src/layouts/sidebar-sidecar/index.tsx

```diff
@@ -108,6 +136,27 @@ const SidebarSidecarLayoutContent = ({
           </div>
         </motion.div>
         <div className={s.contentWrapper}>
+          {currentlyViewedVersion && currentProduct.currentRootDocsPath && (
+            <PageAlert
+              className={s.versionAlert}
+              description={
+                <>
+                  You are viewing documentation for version{' '}
+                  {currentlyViewedVersion}.{' '}
+                  <InlineLink
+                    className={s.versionAlertLink}
+                    href={`/${currentProduct.slug}/${currentProduct.currentRootDocsPath.path}`}
```

Comment:
Can we use the helper from `src/lib/remove-version-from-path.ts`?

Reply:
The intent is that the link should take us to the current path, minus the version. This is how the alert functions now on the io sites. So we would just pass the current path to that function 👍

Reply:
Hmm I'm not sure that's what we want to do. What if the link goes right to a 404 because the route in older docs doesn't exist on the latest version? I think it'd be really frustrating as a user to be linked to a page that doesn't exist.

Reply:
Given that this is going to the _latest_ version, it's very unlikely that the path would not resolve (the inverse is more likely, hence the targeted messaging on the versioned 404 page).

When pages are removed/moved they generally include a redirect, so the latest version will likely resolve. I think it's a nice UX enhancement if users are able to navigate to the latest version of their current page 👍